packages/api/cms-api/src/auth/resolver/auth.resolver.ts

```diff
@@ -33,6 +48,32 @@ export function createAuthResolver(config: AuthResolverConfig): Type<unknown> {
             }
             return signOutUrl;
         }
+
+        @Query(() => Hmac)
+        hmacCreate(): Hmac {
```

Comment:
I don't think those queries should be in AuthResolver. And shouldn't create be a mutation?

Reply:
I refactored it to getSitePreviewHash and moved the resolvers into the PageTreeResolver. Since it doesn't persist anything it should remain a query (like "Hey, just give me the current valid Hash").